fe/fe-core/src/main/java/org/apache/doris/analysis/ExportStmt.java

```diff
@@ -232,16 +232,17 @@ public static String checkPath(String path, StorageBackend.StorageType type) thr
         if (Strings.isNullOrEmpty(path)) {
             throw new AnalysisException("No dest path specified.");
         }
-
+        if (type == StorageBackend.StorageType.BROKER) {
+            if (!path.toLowerCase().contains("bos") && !path.toLowerCase().contains("afs")
+                    && !path.toLowerCase().contains("hdfs")) {
+                throw new AnalysisException("Invalid export path. please use valid 'HDFS://', 'AFS://' or 'BOS://' path.");
+            }
+            return path;
+        }
```

Comment:
How  about a path  `nvalid_schema://xxbos/xxafs_path/path-contians-hdfs`

Reply:
`checkPath` only do a pre-check, it checks the input style.
If the path is not a valid one, an error will be thrown out as expected.
```
 type:ETL_RUN_FAIL; msg:errCode = 2, detailMessage = Broker list path failed.
```

And maybe change `path.contains` to `path.startsWith`, is it better? What's your opinions?